quinn/src/lib.rs

```diff
@@ -788,8 +785,13 @@ impl Future for Driver {
                         packet,
                         ecn,
                     } => {
+                        let destination = if let Some(v4) = destination.ip().to_ipv4() {
```

Comment:
Putting this here means we have to do run through this conversion on every send, which seems a little wasteful? As I understand it, the destination will not change very often over time.

Reply:
Right, per connection it doesn't change, just per send call. I suppose we could switch to storing generic `SocketAddr`s in `Connection`, and only use normalized `SocketAddrV6` for hash table keys, it just feels a little silly when that doesn't actually increase expressiveness any.

Reply:
Maybe, but it seems preferable over converting the address for every send.